sql/core/src/test/resources/sql-tests/inputs/postgreSQL/comments.sql

```diff
@@ -47,4 +45,5 @@ Now just one deep...
 */
 'deeply nested example' AS sixth;
 --QUERY-DELIMITER-END
-/* and this is the end of the file */
+-- [SPARK-30824] Support submit sql content only contains comments.
```

Comment:
Is this an ANSI-related issue? https://issues.apache.org/jira/browse/SPARK-30824

Reply:
No. But postgreSQL test support it.

Reply:
Ah, I see. If so, can you move the parent ticket to https://issues.apache.org/jira/browse/SPARK-30375 ? I think this feature is implementation-dependent.

Reply:
OK.